packages/lsp-server/src/common/features/inlay-hints.ts

```diff
@@ -158,7 +160,7 @@ export class InlayHintFeature implements Feature {
 			if (posting.amount && posting.node) {
 				const amountNode = posting.node.childForFieldName('amount');
 				if (amountNode) {
-					const currencyNode = amountNode.childForFieldName('currency');
+					const currencyNode = amountNode.namedChild(1);
```

Comment:
@fengkx could you explain why this diff is necessary? I don't understand the tree sitter library.